### ISSUE TITLE
Missing docs about chaining `cni` with `cilium`

### ISSUE THREAD

**Issue by mhkarimi1383** [kumahq/kuma#10360]
### What happened?

When we want to chain cni with cilium we have to set

```yaml
cni:
  exclusive: false
```
in helm values

Or setting

```yaml
  cni-exclusive: "false"
```

in Cilium configuration configmap

**Comment by mhkarimi1383** [comment]
@lahabana 
Hi 
I have upgraded my Cilium and I saw that CNI pods are crashing
Then I found that Cilium is watching for changes using fsnotify and then reverts back changes. So Kuma CNI will say that I'm not able to find my configs in the CNI config file

Setting cni-exclusive option fixes the problem

**Comment by michaelbeaumont** [comment]
Yeah, [this option and behavior was added in v1.14](https://github.com/cilium/cilium/releases/tag/v1.14.0), let's update the docs
